core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1156,6 +1158,433 @@ describe('Contributions and review component', () => {
         });
       });
 
+      it('should load reviews, then contributions', fakeAsync(() => {
+        getReviewableTranslationSuggestionsAsyncSpy.and.returnValue(
+          Promise.resolve({
+            suggestionIdToDetails: {
+              suggestion_1: {
+                suggestion: {
+                  target_type: null,
+                  author_name: null,
+                  last_updated_msecs: null,
```

Comment:
Can we provide some sample values instead of `null` where possible?

Reply:
Done